### PR TITLE
Build doc update proposal during doc-walk

### PR DESCRIPTION
**Pull request by desktophero**
## `changes`

`build.md`
- Update `proxy` details in example
- fix links

`sawadm`
- format of public key

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -202,6 +202,10 @@ with Sawtooth installed, and only the run-time dependencies installed.
 These installed images also generate .deb artifacts during build. They can be found
 in the `/tmp` dir in any of the images.
 
+> Internet connection speeds vary. This build step may take some time. 
+
+>There are also possible errors during Docker builds, like transient issues for downloads. An example is the protobuf download from Github step. If the download results in an incomplete/corrupt archive, the build process will exit `non-zero code: 9`
```
Comment (vaporos) [inline BUILD.md]: Please wrap at <80 characters.

Comment (desktophero) [reply BUILD.md]: updated

**Comment by desktophero** [comment]
:cry: 

```
FAILED: User 'desktophero' not whitelisted.
```

**Comment by jsmitchell** [comment]
> 😢
> 
> ```
> FAILED: User 'desktophero' not whitelisted.
> ```

I just kicked off the job for you.

**Comment by rberg2** [comment]
I have added you to the whitelist

**Comment by agunde406** [comment]
I think this PR will need to be rebased to pike up some fixes to handle combining the python sdk with signing.

**Comment by desktophero** [comment]
I've borked my local branch - was working through the `exit code 100` locally and mishandled the local repo. Closing this with a new PR that's up to date.

**Pull request closed without merging**